src/Contracts/Auditable.php

```diff
@@ -9,7 +9,7 @@ interface Auditable
     /**
      * Auditable Model audits.
      *
-     * @return \Illuminate\Database\Eloquent\Relations\MorphMany<OwenIt\Auditing\Models\Audit>
+     * @return \Illuminate\Database\Eloquent\Relations\MorphMany<\OwenIt\Auditing\Models\Audit>
```

Comment:
I think that must be
```diff
-\OwenIt\Auditing\Models\Audit
+\OwenIt\Auditing\Contracts\Audit
```

Reply:
That is not correct. The method returns a model and not the contract.

Reply:
Ok, thanks